src/fixed-card-layout.scss

```diff
@@ -0,0 +1,79 @@
+@import './new-settings';
+@import './mixins';
+
+$block: #{$fd-namespace}-fixed-card-group;
+
+.#{$block} {
+  $fd-border-on-drag: 0.125rem dashed var(--sapContent_DragAndDropActiveColor);
+  $fd-card-margin: 1rem;
+  $fd-card-min-width: 20rem;
+
+  @include fd-reset();
+
+  &--card-layout {
+    @include fd-reset();
+    @include fd-flex();
+
+    justify-content: center;
+
+    * {
+      word-break: break-all;
+    }
+  }
+
+  &--card-layout-column {
+    @include fd-reset();
+
+    flex: 1;
+    margin-right: $fd-card-margin;
+    margin-left: 0;
+
+    @include fd-rtl() {
+      margin-left: $fd-card-margin;
+      margin-right: 0;
+    }
+  }
+
+  &--card-layout-column:last-child {
+    margin-right: 0;
+    margin-left: 0;
+
+    @include fd-rtl() {
+      margin-left: 0;
+      margin-right: 0;
+    }
+  }
+
+  &--card {
+    @include fd-reset();
+
+    margin-bottom: $fd-card-margin;
+    margin-right: 0;
+    min-width: $fd-card-min-width;
+
+    .fd-card__content {
```

Comment:
can't do this

Reply:
removed this now. Earlier card data used to flow, but it has been fixed in card. so not needed.